app/src/main/java/com/jerboa/ui/components/post/PostListings.kt

```diff
@@ -73,32 +74,39 @@ fun PostListings(
                 postView.post.id
             },
         ) { postView ->
-            PostListing(
-                postView = postView,
-                onUpvoteClick = onUpvoteClick,
-                onDownvoteClick = onDownvoteClick,
-                onPostClick = onPostClick,
-                onSaveClick = onSaveClick,
-                onCommunityClick = onCommunityClick,
-                onEditPostClick = onEditPostClick,
-                onDeletePostClick = onDeletePostClick,
-                onReportClick = onReportClick,
-                onPersonClick = onPersonClick,
-                onBlockCommunityClick = onBlockCommunityClick,
-                onBlockCreatorClick = onBlockCreatorClick,
-                isModerator = false,
-                showCommunityName = showCommunityName,
-                fullBody = false,
-                account = account, // TODO can't know with many posts
-                postViewMode = postViewMode,
-                showVotingArrowsInListView = showVotingArrowsInListView,
-                enableDownVotes = enableDownVotes,
-                showAvatar = showAvatar,
-                useCustomTabs = useCustomTabs,
-                usePrivateTabs = usePrivateTabs,
-                blurNSFW = blurNSFW,
-            )
-            Divider(modifier = Modifier.padding(bottom = SMALL_PADDING))
+            Card(
```

Comment:
For the list view mode, we might not want it to be surrounded in a card.

Reply:
Now with the smaller padding, the effect is fairly subtle: https://github.com/dessalines/jerboa/assets/1407980/c92a039c-db49-4f50-a7f9-7580002759fd

Is it okay to keep this as-is? It'd be one less code path to maintain.

Reply:
With the title font size customizable again, it's not too different vertical space-wise from what I can tell.

Reply:
True, it's not too different from having the divider. As for the padding of the arrows, it looks like the alignment of the composables was already correct:

![image](https://github.com/dessalines/jerboa/assets/1407980/88823079-8fcf-4385-bbaa-f003dbfdbfc9)

I'm not sure if I can control the padding at the top of the text, so leaving it as-is for now.